pygame/compat.py

```diff
@@ -114,3 +114,26 @@ def next_(i, *args):
     import itertools.imap as imap_
 except ImportError:
     imap_ = map
+
+
+# The behaviour of the power operator, '**' and built-in pow() function,
+# changed in Python 3.x.  In Python 3.x raising a negative number to a
+# fractional power results in a complex number, while in earlier versions
+# it raised a ValueError.  Matching the behaviour of pygame requires
+# the ValueError instead of a complex result.  The compatibility function
+# defined below is provided for this purpose.
```

Comment:
I tried `math.pow()`, but there's an issue with `0 ** -1`, which raises a `ZeroDivisionError`, while the math module raises a `ValueError`.  The unit tests have a check or the former.
Interesting point about the implementation differences.